pkg/ingester/ingester.go

```diff
@@ -79,7 +80,7 @@ func (i *ingesterFlusherCompat) Flush() {
 	}
 }
 
-func New(phlarectx context.Context, cfg Config, dbConfig phlaredb.Config, storageBucket phlareobj.Bucket, limits Limits) (*Ingester, error) {
+func New(phlarectx context.Context, cfg Config, dbConfig phlaredb.Config, storageBucket phlareobj.Bucket, limits Limits, queryStoreAfter time.Duration) (*Ingester, error) {
```

Comment:
Is this the best way to get access to the `querier.query_store_after` configuration? I can't pass the config in directly here because the `querier` package already depends on the `ingester` package and that would create a cyclic dependency.

Reply:
cc @cyriltovena

Reply:
May you can do this config validation outside of ingester ? 

```
if queryStoreAfter > 0 {
		retentionPolicy.Expiry = queryStoreAfter
	}
```

In the config validation or the `phlare` package. It's not great but it's also alright.